index.html

```diff
@@ -358,9 +362,9 @@ <h3>StatusList2021Entry</h3>
               <td>statusListCredential</td>
               <td>
 The <code>statusListCredential</code> property MUST be a URL to a
-<a>verifiable credential</a>. When the URL is dereferenced, the resulting
-<a>verifiable credential</a> MUST have <code>type</code> property that
-includes the <code>StatusList2021Credential</code> value.
+<a>verifiable credential</a>. When the URL is dereferenced, the result
+MUST be a <a>verifiable credential</a> that contains a <code>type</code>
```

Comment:
```suggestion
MUST be a <a>Verifiable Credential</a> that contains a <code>type</code>
```

Reply:
The VCWG decided on uncapitalized terminology throughout the vc-data-model specification (example here):

https://www.w3.org/TR/vc-data-model/#ecosystem-overview

We had started with capitalized terms and folks complained that the prose was harder to read, so we switched to lower case terms.

Reply:
I'm resolving this, given the thumbs up from OP.